### DISCUSSION TITLE
Sounds amazing...for a developer to build, not for somebody who can actually use it

### DISCUSSION THREAD

**Discussion by rwnq8** [BerriAI/litellm#3643] [General]
Sorry you're the first on my list of rants that here I am trying to make democratizing access to knowledge as easy as Google made the search box and I keep banging my head against the wall with all these brilliant solutions like yours that fundamentally miss the point of who's going to use this and just giving more work to developers or AI hacks is just not really all that useful.

**Comment by ishaan-jaff** [comment]
Thanks, what extra work did litellm create for you? Happy to address them @RwnQ8

**Reply by rwnq8** [reply]
> Thanks, what extra work did litellm create for you? Happy to address them @RwnQ8 

Here's the thing I would like to encourage dev teams to consider from my perspective going from product to end-user is that we tend to get bogged down on these things that have amazing potential. I've looked at dozens if not hundreds of solutions to try to make it easy for anyone to get access to the same diversity of models Hugging Face is starting to do with their HF Chat, but I haven't actually seen a lot of end-user ready solutions even though the stuff underneath it is absolutely amazing so I wanna give you guys credit for that and challenge you to not get too hung up in the weeds because is the screenshot I'm seeing on help page about billing and cost management. I feel like we've really missed the boat on that one when the headline should be that this is the most amazing thing since the printing press and language itself available to everyone basically for free.

**Reply by ishaan-jaff** [reply]
@RwnQ8 I don't follow. Can you give me a bullet point list of extra work litellm created for you or bugs you faced? I will address all your issues by the end of the week
